raiden/network/utils.py

```diff
@@ -1,47 +1,47 @@
-from itertools import count
+import errno
+import socket
+from itertools import count, repeat
+from socket import SocketKind
 from time import sleep
 from typing import Optional
 
-import psutil
 import requests
 from requests import RequestException
 
 
-def get_free_port(address: str, initial_port: int):
-    """Find an unused TCP port in a specified range. This should not
-      be used in misson-critical applications - a race condition may
-      occur if someone grabs the port before caller of this function
-      has chance to use it.
-      Parameters:
-          address : an ip address of interface to use
-          initial_port : port to start iteration with
-      Return:
-          Iterator that will return next unused port on a specified
-          interface
+def get_free_port(
+    bind_address: str = '127.0.0.1',
+    initial_port: int = 0,
```

Comment:
Do we really want to begin looking for a free port at 0? Maybe use a higher default value like 1024.